### DISCUSSION TITLE
Windows下如何结合Burpsuite使用？

### DISCUSSION THREAD

**Discussion by vincentTea** [xjasonlyu/tun2socks#189] [Q&A]
我想抓包某个应用，已经他会跟https://www.xxx.com进行通讯。
我的做法是这样：

1、安装Burpsuite的证书
2、开启Burpsuite代理：指定本机网卡192.168.31.100:8080
3、ping www.xxx.com得到ip地址，如： 1.2.3.4
3、使用tun2socks

`tun2socks -device wintun -proxy http://192.168.31.100:8080`
`netsh interface ip set address name="wintun" source=static addr=192.168.123.1 mask=255.255.255.0 gateway=none`

4、将第三步得到的ip或ip段使用route到wintun上去

route print，得到
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
接口列表
  51..............................................WireGuard Tunnel （tun2socks的)
   6...7Z 8G 6F 52 14 FE .......Realtek PCIe GBE Family Controller (本地物理网卡)
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - 

执行以下，将ip段挂过去
`route add 1.2.3.0 mask 255.255.255.0  192.168.123.1 if 6 metric 5`

**然后发现，该网站不能正常上了。。**
其实这里有两个地方我不太明白：
在wiki指导中的windows最后两行，我没明白：

route add 0.0.0.0 mask 0.0.0.0 192.168.123.1 if <IF NUM> metric 5
这一句我的理解是：将本地所有网段的通讯都通过192.168.123.1网络适配器转发，也就是tun2socks创建的那个
但这里的：
<IF NUM> 是指 tun2socks的接口号，还是实际物理网卡的接口号

route add <server ip> mask 255.255.255.255 <primary gateway ip>
这一句我的理解是：将server ip 指定由 primary gateway ip的网络适配器转发
这里的server ip我不太理解是指什么，是原本我上述所写的：本机网卡 192.168.31.100 这个ip吗
primary gateway ip 是指网我原本的网关(192.168.31.1)，还是192.168.31.100

希望有大佬不吝指教，谢谢

**Comment by yundmb** [comment]
请问，问题解决了吗？
